types/Modal.svelte.d.ts

```diff
@@ -22,10 +19,11 @@ export type Styles = Record<string, string | number>;
 
 export type TransitionFn = (
   node: Element,
-  parameters: BlurParams | FadeParams | FlyParams | SlideParams
```

Comment:
Can you give me write access to your to your PR so I can fix the removal of these types?